rules/sublime_rules/sublime_mailboxes_deactivated.py

```diff
@@ -0,0 +1,10 @@
+from panther_sublime_helpers import sublime_alert_context
+
+
+def rule(event):
+    all_events = event.deep_walk("events", "type")
```

Comment:
This will likely need to be rewritten without `deep_walk`, as per my other comment each item in the `events` array is actually being treated as a separate log.